holoviews/plotting/mpl/chart3d.py

```diff
@@ -33,6 +33,9 @@ class Plot3D(ColorbarPlot):
     bgcolor = param.String(default='white', doc="""
         Background color of the axis.""")
 
+    labelled = param.List(default=['x', 'y', 'z'], doc="""
+        Specifies the list of axes to display the labels on.""")
+
```

Comment:
Minor docstring improvement. Should probably read "Whether to plot the 'x', 'y' and 'z' labels" for consistency.